### ISSUE TITLE
Implementar carga/capacidad

### ISSUE THREAD

**Issue by pedrobaeza** [odoomrp/odoomrp-wip#697]
Pongo este issue como referencia para cuando implementemos esta funcionalidad.

**Comment by pedrobaeza** [comment]
Comienzo con dos módulos que me ha apuntado David Beal de Akretion:

https://github.com/akretion/manufacturing/tree/7.0/mrp_workcenter_load
https://github.com/akretion/manufacturing/tree/7.0/mrp_load_by_schedule_state

**Comment by anajuaristi** [comment]
@pedrobaeza entiendo que la primera tarea es migrar esos 2 módulos a la V8. 
Si es eso asigno a @Daniel-CA para que vaya migrándolos. Sino, coméntanos más específicamente lo que quieres que se haga...

**Comment by pedrobaeza** [comment]
No, la primera tarea es probarlos en v7 y ver qué parte interesa.
